--- pgrx-pg-config-0.16.1/Cargo.toml	1970-01-01T00:00:01+00:00
+++ pgrx-pg-config-0.16.1/Cargo.toml	2025-10-23T08:04:32.388348+00:00
@@ -61,12 +61,3 @@
 [dependencies.url]
 version = "2.5.4"
 
-[target.'cfg(target_os = "windows")'.dependencies.codepage]
-version = "0.1.2"
-
-[target.'cfg(target_os = "windows")'.dependencies.encoding_rs]
-version = "0.8.35"
-
-[target.'cfg(target_os = "windows")'.dependencies.winapi]
-version = "0.3.9"
-features = ["winnls"]
