From f9610c38c307a4ebd3160932322e4d4268c08c17 Mon Sep 17 00:00:00 2001
From: Mat Booth <mat.booth@gmail.com>
Date: Tue, 28 Oct 2025 16:44:22 +0000
Subject: [PATCH 2/3] fix mismatched types in if expression

---
 src/index/hook.rs | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/index/hook.rs b/src/index/hook.rs
index bfada43..7a5fa5a 100644
--- a/src/index/hook.rs
+++ b/src/index/hook.rs
@@ -128,10 +128,10 @@ unsafe extern "C-unwind" fn executor_start(
         use pgrx::pg_sys::submodules::ffi::pg_guard_ffi_boundary;
         if let Some(prev_executor_start) = PREV_EXECUTOR_START {
             #[allow(ffi_unwind_calls, reason = "protected by pg_guard_ffi_boundary")]
-            pg_guard_ffi_boundary(|| prev_executor_start(query_desc, eflags))
+            pg_guard_ffi_boundary(|| prev_executor_start(query_desc, eflags));
         } else {
-            pgrx::pg_sys::standard_ExecutorStart(query_desc, eflags)
-        }
+            pgrx::pg_sys::standard_ExecutorStart(query_desc, eflags);
+        };
         pg_guard_ffi_boundary(|| rewrite_plan_state((*query_desc).planstate, null_mut()));
     }
 }
-- 
2.51.1

